pkg/sentry/certs/certs.go

```diff
@@ -1,6 +1,7 @@
 package certs
 
 import (
+	"crypto"
```

Comment:
@ItalyPaleAle can this file change alone be moved to a separate PR? 
And only have at most tests and 1.20 change in this PR?

Reply:
Sure, done.